x/ecocredit/server/core/features/msg_allowed_class_creator.feature

```diff
@@ -0,0 +1,89 @@
+Feature: Msg/AllowedClassCreator
+
+  A class creator can be added:
+  - when the class creator does not exist
+  - when the authority is a governance account address
+  - the class creator is added
+
+  Rule: The class creator does not exist
+
+    Scenario: The class creator does not exist
+      When alice attempts to add class creators with properties
+      """
+      {
+        "authority":"regen1nzh226hxrsvf4k69sa8v0nfuzx5vgwkczk8j68",
+        "creators":[
+            "regen1nzh226hxrsvf4k69sa8v0nfuzx5vgwkczk8j68"
+        ]
+      }
+      """
+      Then expect no error
```

Comment:
could we check resulting state here?

Reply:
We should be checking state but in a separate rule for `the class creator is added`. We want to check no error and error within a rule, and state within a separate rule specific to checking state was updated. This is correct here but looks like we are missing a rule and scenario for `the class creator is added` list above.